### PR TITLE
[MASTERDIET-60] Login form adjustment

### PR DESCRIPTION
**Pull request by jakubowiczish**

### CODE REVIEW

File: src/user/login/LoginSignupContainer.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+    const loginRequest = {
+      'email': this.state.loginEmail,
+      'password': this.state.loginPassword
+    }
+
+    login(loginRequest)
+      .then(response => {
+        localStorage.setItem(ACCESS_TOKEN, response.accessToken)
+        // this.props.history.push('/')
```
Comment (jfoder) [inline src/user/login/LoginSignupContainer.js:60]: Should be removed

File: src/user/login/LoginSignupContainer.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                <a href={GITHUB_AUTH_URL} className="social fab github-icon"/>
+              </div>
+              <span>or use your account</span>
+              <input type="email"
+                     placeholder="Email"
+                     name="loginEmail"
+                     value={this.state.loginEmail}
+                     onChange={this.handleInputChange} required
+              />
+              <input type="password"
```
Comment (jfoder) [inline src/user/login/LoginSignupContainer.js:137]: Can we add some borders to these inputs?

**Review by jfoder — CHANGES_REQUESTED** [review]

File: src/user/login/LoginSignupContainer.js (hunk 1 above)
Comment (jakubowiczish) [reply src/user/login/LoginSignupContainer.js]: Fixed

File: src/user/login/LoginSignupContainer.js (hunk 2 above)
Comment (jakubowiczish) [reply src/user/login/LoginSignupContainer.js]: Fixed

**Review by jfoder — APPROVED** [review]

**Pull request merged**